### ISSUE TITLE
Adding a capacitor says "capacitor loop with no resistance!"

### ISSUE THREAD

**Issue by Killeroid17** [sharpie7/circuitjs1#845]
Have been always adding capacitors in parallel with power supplies or in circuit anything else, and never had this error.
tried to modify all component's values but always the same...
this as an example of simplest simulation cannot be done not to mention more advanced ones...

**Please** put an 'Archived older versions' in the 'Standalone (offline) versions.' section...

![image](https://user-images.githubusercontent.com/34105827/218254129-c174fa63-2a95-4b57-9a1d-175bcf42bff0.png)

**Comment by pfalstad** [comment]
This has been there forever.  See #634.

**Comment by Killeroid17** [comment]
> This has been there forever. See #634.

yes already wasted my time trying SEVA77's older version's and yes it behaves the same way,
But I know long time ago I applied in parallel a cap and didn't had problems I mean I've used caps in any combination without this error,
can't this be implemented so it's general wires to have it's own resistance ? simulating this in real life model won't give you any *SPARK*... so what can you do about it to avoid getting error on this trivial connection ?

**Comment by pfalstad** [comment]
Changing wires to have their own resistance would slow down the simulator quite a lot.

I could probably fix this, I will look into it some more.

**Comment by Killeroid17** [comment]
> Changing wires to have their own resistance would slow down the simulator quite a lot.
> 
> I could probably fix this, I will look into it some more.

or at least remove the error and leave the project running without being halted / stopped

**Comment by esaruoho** [comment]
Fixed in v3-dev — capacitor loop error removed, capacitor series resistance added.
